### PR TITLE
refactor(angular): accordion merge

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/angular/accordion/expansion-panel-header/expansion-panel-header.scss [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+$sbbExpansionPanelHeaderLineHeightStandard: 28;
+$sbbExpansionPanelHeaderLineHeightStandardTablet: 30;
+$sbbExpansionPanelHeaderLineHeightStandard4k: 45;
+$sbbExpansionPanelHeaderLineHeightStandard5k: 60;
+
+.sbb-expansion-panel-header {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/angular/accordion/expansion-panel-header/expansion-panel-header.scss:11]: it's not necessary, but personally I like to define it explicitly. Will discuss it with you :)

File: src/components-examples/angular/accordion/accordion-custom-html/accordion-custom-html-example.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ */
+@Component({
+  selector: 'sbb-accordion-custom-html-example',
+  templateUrl: './accordion-custom-html-example.html',
+  styleUrls: ['./accordion-custom-html-example.css'],
+})
+export class AccordionCustomHtmlExample {
+  logAndPreventOpeningPanel(evt: any) {
+    console.log('Button clicked and prevented panel to open');
+    evt.preventDefault();
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components-examples/angular/accordion/accordion-custom-html/accordion-custom-html-example.ts:15]: stopPropagation only is ok

File: src/angular/accordion/expansion-panel/expansion-panel.scss [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+.sbb-expansion-panel-header {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  position: relative;
+  background-color: $sbbColorMilk;
+  outline: none;
+  padding: pxToRem(23 - $sbbExpansionPanelBorderWidth) pxToRem(23 - $sbbExpansionPanelBorderWidth)
+    pxToRem(25 - $sbbExpansionPanelBorderWidth) pxToRem(23 - $sbbExpansionPanelBorderWidth);
```
Comment (kyubisation) [inline src/angular/accordion/expansion-panel/expansion-panel.scss:66]:
```suggestion
  padding: pxToRem(24 - $sbbExpansionPanelBorderWidth) pxToRem(24 - $sbbExpansionPanelBorderWidth)
    pxToRem(26 - $sbbExpansionPanelBorderWidth) pxToRem(24 - $sbbExpansionPanelBorderWidth);
```

File: src/angular/accordion/expansion-panel/expansion-panel.scss [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+  &.sbb-expanded {
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      height: 0;
+      bottom: 0;
+      width: calc(100% - #{pxToRem((24 - $sbbExpansionPanelBorderWidth) * 2)});
+      left: pxToRem((24 - $sbbExpansionPanelBorderWidth));
+      border-bottom: pxToRem(1) solid $sbbColorCloud;
```
Comment (kyubisation) [inline src/angular/accordion/expansion-panel/expansion-panel.scss:104]:
```suggestion
      border-bottom: pxToRem($sbbExpansionPanelBorderWidth) solid $sbbColorCloud;
```

File: src/angular/accordion/expansion-panel/expansion-panel.scss [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+          left: pxToRem((48 - $sbbExpansionPanelBorderWidth));
+        }
+        @include mq($from: desktop) {
+          width: calc(100% - #{pxToRem((32 - $sbbExpansionPanelBorderWidth) * 2)});
+          left: pxToRem((32 - $sbbExpansionPanelBorderWidth));
+        }
+        @include mq($from: desktop4k) {
+          width: calc(100% - #{pxToRem((48 - $sbbExpansionPanelBorderWidth4k) * 2)});
+          left: pxToRem((48 - $sbbExpansionPanelBorderWidth4k));
+          border-width: pxToRem(2);
```
Comment (kyubisation) [inline src/angular/accordion/expansion-panel/expansion-panel.scss:118]:
```suggestion
          border-width: pxToRem($sbbExpansionPanelBorderWidth4k);
```

**Pull request merged**